tket/src/Predicates/PassGenerators.cpp

```diff
@@ -548,6 +548,22 @@ PassPtr KAKDecomposition(double cx_fidelity) {
   return std::make_shared<StandardPass>(precons, t, postcon, j);
 }
 
+PassPtr DecomposeTK2() { return DecomposeTK2({}); }
+PassPtr DecomposeTK2(const Transforms::TwoQbFidelities& fid) {
+  Transform t = Transforms::decompose_TK2(fid);
+  const PredicatePtrMap no_precons;
+  PredicateClassGuarantees preserve_all;
+  PostConditions postcons{{}, preserve_all, Guarantee::Preserve};
+  nlohmann::json j;
+  j["name"] = "DecomposeTK2";
+  nlohmann::json fid_json;
+  fid_json["CX"] = fid.CX_fidelity;
+  fid_json["ZZPhase"] = "SERIALIZATION OF FUNCTIONS IS NOT YET SUPPORTED";
```

Comment:
```suggestion
  fid_json["ZZPhase"] = "SERIALIZATION OF FUNCTIONS IS NOT SUPPORTED";
```
Not sure if it will ever be.

Reply:
Done.